modules/op-release-notes-1-8.adoc

```diff
@@ -12,13 +12,198 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+// (link:https://issues.redhat.com/browse/SRVKP-2250[SRVKP-2250 Install Hub with default configs])
+// Shiv Verma
+
+// [Chains]
+* Previously, the `chains-config` config map for {tekton-chains} in the `openshift-pipelines` namespace was automatically reset to default after upgrading the {pipelines-title} Operator. The current update fixes this issue. Now, you can configure key-value pairs in the `chains-config` config map, and those values persist after upgrading the {pipelines-title} Operator.
+// https://issues.redhat.com/browse/SRVKP-2349
+// Piyush Garg @piyush-garg
+
+* This update adds `GIT_SSL_CAINFO` to the `git-clone` task so you can clone secured repositories.
+// https://issues.redhat.com/browse/SRVKP-1423
+// @savitaashture
+
+[id="hub-new-features-1-8_{context}"]
+=== Hub
+
+* With this update, you can disable the login and rating features of your {tekton-hub} web console. To do so, omit adding a value for the `AUTH_BASE_URL` field in the `<tekton_hub_repository>/config/02-api/20-api-secret.yaml` file.
+// https://github.com/tektoncd/hub/pull/542
+// Shiv Pratap Verma @pratap0007
+
+* With this update, you don't need to log in to the {tekton-hub} web console to add resources from the catalog to the database. Now, these resources are automatically added when the catalog refresh API runs.
+// https://github.com/tektoncd/hub/pull/510
+// Shiv Pratap Verma @pratap0007
+
+* This update automatically refreshes the catalog every 30 minutes by calling the catalog refresh API job. This interval is user-configurable.
+// https://github.com/tektoncd/hub/pull/547
+// Shiv Pratap Verma @pratap0007
+
+
+[id="breaking-changes-1-8_{context}"]
+== Breaking changes
+
+// .Pipelines
+
+* This update changes the the default metrics level of `TaskRun` and `PipelineRun` resources to the following values:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-observability
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipelines
+data:
+  _example: |
+  ...
+    metrics.taskrun.level: "task"
+    metrics.taskrun.duration-type: "histogram"
+    metrics.pipelinerun.level: "pipeline"
+    metrics.pipelinerun.duration-type: "histogram"
+----
+// link:https://github.com/tektoncd/pipeline/pull/4630
+
+* With this update, if an annotation or label is present in both `Pipeline` and `PipelineRun` resources, the value in the `Run` type takes precedence. The same is true if an annotation or label is present in `Task` and `TaskRun` resources.
+// link:https://github.com/tektoncd/pipeline/pull/4638
+// Vincent Demeester
+
+* In {pipelines-title} 1.8, the previously deprecated `PipelineRun.Spec.ServiceAccountNames` field has been removed. Action required: Use the `PipelineRun.Spec.TaskRunSpecs` field instead.
```

Comment:
I assume you're using `Action required:` as an internal note to yourself?

Reply:
Removed three instances of `Action required:`. Thanks.